README.md

```diff
@@ -20,3 +21,26 @@ Then, add this to your `~/.bash_profile` to make it permanent:
 ```
 source ~/.rvm/scripts/rvm
 ```
+
+
+## Publishing An Episode
+Install the dependencies:
+```bash
+cd scripts
+pip3 install -r requirements.txt
+```
+
+```bash
+cd scripts
+./create-pull-request [episode-number]
+```
+
+Use the trello helper scripts to generate the body of the PR, the URL, and the title:
+```bash
+cd scripts
+./trello-episode-title [episode-number]
+./trello-episode-url [episode-number]
+./trello-episode-description [episode-number]
```

Comment:
You don't have to do any of this manually. The `upload` script calls all these scripts to do its job. To publish a new episode, you **only** have to use the `upload` script.